### ISSUE TITLE
Perform kappa, sigma clipping using multiple CPU cores. Try Ray.

### ISSUE THREAD

**Issue by HannoSpreeuw** [transientskp/pyse#11]
Usually, a large image is divided into many subimages because background statistics can vary over the image.
The background statistics are derived by kappa, sigma clipping.
Presently, this is done serially, so subimage after subimage.

Since this is an "embarassingly parallel" problem, so the output of one subimage does not depend on another, we can use all the cores of a CPU simultaneously.

Ray is just one package we can try.

**Comment by HannoSpreeuw** [comment]
I created the ` Try_Ray_for_multithreaded_kappa_sigma_clipping_of_subimages`  branch and I achieved a factor 5 speedup for kappa, sigma clipping on a Intel Core i7-4790K CPU @ 4.00GHz, which has four physical and eight logical cores.

**Comment by HannoSpreeuw** [comment]
This branch can be merged with the `accelerated` branch after fixing this Travis build problem that occurs when building the Docker image:
```
E: Unable to locate package python-ray

E: Unable to locate package python3-ray
```

So this comes from the Dockerfile:
```
FROM kernsuite/base:3
RUN docker-apt-install \
```
So apparently Ray is not part of kernsuite/base:3.

@gijzelaerr could you help me out?

**Comment by HannoSpreeuw** [comment]
Fixed by #12.
